--- a/Makefile.am
+++ b/Makefile.am
@@ -13,7 +13,8 @@ AM_LDFLAGS = \
 data_DATA =
 
 bin_PROGRAMS = \
-	lsusb
+	lsusb \
+	usbreset
 
 sbin_SCRIPTS =
 
@@ -21,9 +22,6 @@ bin_SCRIPTS = \
 	usb-devices \
 	lsusb.py
 
-noinst_PROGRAMS = \
-	usbreset
-
 lsusb_SOURCES = \
 	lsusb.c lsusb.h \
 	lsusb-t.c \
@@ -42,9 +40,6 @@ lsusb_LDADD = \
 	$(LIBUSB_LIBS) \
 	$(UDEV_LIBS)
 
-usbreset_SOURCES = \
-	usbreset.c
-
 man_MANS = \
 	lsusb.8	\
 	usbhid-dump.8 \
